dev/flow-typed-guardian-frontend/ab-tests.js

```diff
@@ -2,15 +2,15 @@
 
 type ListenerFunction = (f: () => void) => void;
 
-export type Variant = {
+declare type Variant = {
```

Comment:
is a global type called `Variant` clear outside the test context? it will be available elsewhere, maybe `ABTestVariant`?